src/tests/comment.test.js

```diff
@@ -0,0 +1,213 @@
+import app from '../index';
+import db from '../database/models';
+import GenerateToken from '../helpers/token.helper';
+import {
+  roomDatabaseData,
+  accomodationDatabaseData,
+  locationDatabaseData,
+  UserDatabaseData,
+  userManagementDatabaseData,
+  accomodationTypeDatabaseData,
+  comment, trip, requestTrip
+} from './mock/comment.mock.data';
+
+chai.use(chaiHttp);
+chai.should();
+
+const token = GenerateToken({ email: 'user@gmail.com', isVerified: 'true' });
+const anauthorizedToken = GenerateToken({ email: 'unauthorized@gmail.com', isVerified: 'true' });
+const linemanagerToken = GenerateToken({ email: 'linemanager@gmail.com', isVerified: 'true' });
+
+
+describe('comment tests', () => {
+  before(async () => {
+    await db.user.destroy({ where: {}, force: true });
+    await db.user.create(UserDatabaseData[0]);
+    await db.user.create(UserDatabaseData[1]);
+    await db.user.create(UserDatabaseData[2]);
+    await db.locations.create(locationDatabaseData[0]);
+    await db.locations.create(locationDatabaseData[1]);
+    await db.locations.create(locationDatabaseData[2]);
+    await db.locations.create(locationDatabaseData[3]);
+    await db.accomodationtype.create(accomodationTypeDatabaseData[0]);
+    await db.accomodationtype.create(accomodationTypeDatabaseData[1]);
+    await db.accomodationtype.create(accomodationTypeDatabaseData[2]);
+    await db.accomodation.create(accomodationDatabaseData[0]);
+    await db.accomodation.create(accomodationDatabaseData[1]);
+    await db.accomodation.create(accomodationDatabaseData[2]);
+    await db.accomodation.create(accomodationDatabaseData[3]);
+    await db.rooms.create(roomDatabaseData[0]);
+    await db.rooms.create(roomDatabaseData[1]);
+    await db.rooms.create(roomDatabaseData[2]);
+    await db.rooms.create(roomDatabaseData[3]);
+    await db.usermanagement.create(userManagementDatabaseData[0]);
+    await db.trips.create(trip[0]);
+    await db.trips.create(trip[1]);
+    await db.requesttrip.create(requestTrip[0]);
+    await db.requesttrip.create(requestTrip[1]);
+    await db.comment.create(comment[0]);
+    await db.comment.create(comment[1]);
+  });
+
+  it('comment should be created successfuly', (done) => {
+    chai
+      .request(app)
+      .post(`/api/v1/trip-request/${trip[0].tripId}/comments`)
+      .set('token', `Bearer ${token}`)
+      .send({ comment: 'new comment' })
+      .end((err, res) => {
+        res.should.have.status(201);
+        res.body.should.have.be.a('object');
+        res.body.should.have.property('message').eql('comment created successfuly');
+        res.body.data.should.have.property('id');
+        res.body.data.should.have.property('subjectId');
+        res.body.data.should.have.property('subjectType');
+        res.body.data.should.have.property('commentorId');
+        res.body.data.should.have.property('comment');
+        res.body.data.should.have.property('updatedAt');
+        res.body.data.should.have.property('createdAt');
```

Comment:
@shemaeric1234 It's good that you're asserting if the response object has certain properties but you should also check the expected data or property values, update elsewhere it's applicable.